env.go.jp,"HTTPSConnectionPool(host='env.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face6b23f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='env.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face66c81c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
epm.com.co,"HTTPSConnectionPool(host='epm.com.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face4150100>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='epm.com.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face69da550>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
eprice.it,"HTTPSConnectionPool(host='www.eprice.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.eprice.it', port=443): Read timed out. (read timeout=2)"
equipement.gov.ma,"HTTPSConnectionPool(host='equipement.gov.ma', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face6541c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='equipement.gov.ma', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face4f1c790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
erar.si,"HTTPSConnectionPool(host='erar.si', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='erar.si', port=443): Read timed out. (read timeout=2)"
erasmusmc.nl,"HTTPSConnectionPool(host='erasmusmc.nl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='erasmusmc.nl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
erbakan.edu.tr,"HTTPSConnectionPool(host='erbakan.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='erbakan.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
erciyes.edu.tr,"HTTPSConnectionPool(host='erciyes.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='erciyes.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
eroski.es,"HTTPSConnectionPool(host='eroski.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='eroski.es', port=443): Read timed out. (read timeout=2)"
esaunggul.ac.id,"HTTPSConnectionPool(host='esaunggul.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='esaunggul.ac.id', port=443): Read timed out. (read timeout=2)"
escuelasoulsurf.cl,"HTTPSConnectionPool(host='escuelasoulsurf.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face5d789a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='escuelasoulsurf.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face2b66220>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
eskom.co.za,"HTTPSConnectionPool(host='eskom.co.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face26e1d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='eskom.co.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face2c291c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
esl-lounge.com,"HTTPSConnectionPool(host='esl-lounge.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='esl-lounge.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
eso.lt,"HTTPSConnectionPool(host='eso.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='eso.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
esocietyplatform.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
espe.edu.ec,"HTTPSConnectionPool(host='espe.edu.ec', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face2697730>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='espe.edu.ec', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face4b67940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
espoch.edu.ec,"HTTPSConnectionPool(host='espoch.edu.ec', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='espoch.edu.ec', port=443): Read timed out. (read timeout=2)"
espol.edu.ec,"HTTPConnectionPool(host='www.espol.edu.ec', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.espol.edu.ec', port=80): Read timed out. (read timeout=2)"
esteri.it,"HTTPSConnectionPool(host='esteri.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face0f5d1f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='esteri.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face5060520>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
etimad.sa,"HTTPSConnectionPool(host='etimad.sa', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='etimad.sa', port=443): Read timed out. (read timeout=2)"
etmall.com.tw,"HTTPSConnectionPool(host='etmall.com.tw', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'etmall.com.tw' doesn't match 'www.etmall.com.tw'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
etno-muzej.si,"HTTPSConnectionPool(host='etno-muzej.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face0a386a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='etno-muzej.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face1381490>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
etstur.com,"HTTPSConnectionPool(host='www.etstur.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.etstur.com', port=443): Read timed out. (read timeout=2)"
eureka.org.il,"HTTPSConnectionPool(host='eureka.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='eureka.org.il', port=443): Read timed out. (read timeout=2)"
euro.com.pl,"HTTPSConnectionPool(host='www.euro.com.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.euro.com.pl', port=443): Read timed out. (read timeout=2)"
euro.cz,"HTTPSConnectionPool(host='www.euro.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.euro.cz', port=443): Read timed out. (read timeout=2)"
eurocartv.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
eurolines.fr,"HTTPSConnectionPool(host='eurolines.fr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face4c163a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='eurolines.fr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face0d46550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
evidus.com,"HTTPSConnectionPool(host='evidus.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7facdf5d37f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='evidus.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face4a85d60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
evkur.com.tr,"HTTPSConnectionPool(host='evkur.com.tr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7facdf1b9820>, 'Connection to evkur.com.tr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='evkur.com.tr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7face4d5ae50>, 'Connection to evkur.com.tr timed out. (connect timeout=2)'))"
evz.ro,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
excelsior.com.mx,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
expatchoice.asia,"HTTPSConnectionPool(host='www.expatchoice.asia', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.expatchoice.asia', port=443): Read timed out. (read timeout=2)"
extra.com.br,"HTTPSConnectionPool(host='extra.com.br', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.extra.com.br', port=80): Read timed out. (read timeout=2)"
extrafarma.com.br,"HTTPSConnectionPool(host='extrafarma.com.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face41984f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='extrafarma.com.br', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7facdcefbf40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
extrasports.com,Exceeded 30 redirects.,Exceeded 30 redirects.
fagbokforlaget.no,"HTTPSConnectionPool(host='fagbokforlaget.no', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='fagbokforlaget.no', port=443): Read timed out. (read timeout=2)"
fajar.co.id,"HTTPSConnectionPool(host='fajar.co.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='fajar.co.id', port=443): Read timed out. (read timeout=2)"
famigliacristiana.it,"HTTPSConnectionPool(host='famigliacristiana.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face62b49d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='famigliacristiana.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7facdc6e5760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
family.co.jp,"HTTPSConnectionPool(host='family.co.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7face5ddfeb0>, 'Connection to family.co.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='family.co.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7facdaee5580>, 'Connection to family.co.jp timed out. (connect timeout=2)'))"
famitsu.com,"HTTPSConnectionPool(host='famitsu.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='famitsu.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
fans.lv,"HTTPSConnectionPool(host='www.fans.lv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.fans.lv', port=443): Read timed out. (read timeout=2)"
farbotex.it,"HTTPSConnectionPool(host='farbotex.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'farbotex.it' doesn't match either of '*.aruba.it', 'aruba.it'"")))","HTTPConnectionPool(host='www.farbotex.it', port=80): Read timed out. (read timeout=2)"
farfetch.com,"HTTPSConnectionPool(host='www.farfetch.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.farfetch.com', port=443): Read timed out. (read timeout=2)"
farmaciasguadalajara.com,"HTTPSConnectionPool(host='farmaciasguadalajara.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face660eb50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='farmaciasguadalajara.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face48ae460>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
farmaciasportuguesas.pt,"HTTPSConnectionPool(host='farmaciasportuguesas.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face1498580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='farmaciasportuguesas.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face499d760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
farmaku.com,"HTTPSConnectionPool(host='www.farmaku.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.farmaku.com', port=443): Read timed out. (read timeout=2)"
farmashop.com.uy,"HTTPSConnectionPool(host='farmashop.com.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='tienda.farmashop.com.uy', port=443): Read timed out. (read timeout=2)"
farmhopping.com,"HTTPSConnectionPool(host='farmhopping.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='farmhopping.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
fashion-press.net,"HTTPSConnectionPool(host='fashion-press.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7face45efc70>, 'Connection to fashion-press.net timed out. (connect timeout=2)'))","HTTPConnectionPool(host='fashion-press.net', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7face499dc70>, 'Connection to fashion-press.net timed out. (connect timeout=2)'))"
fashionismo.com.br,"HTTPSConnectionPool(host='fashionismo.com.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7facdc0f82e0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPSConnectionPool(host='fashionismo.com.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7facdf5d30a0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
fassil.com.bo,"HTTPSConnectionPool(host='fassil.com.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.fassil.com.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
fazenda.gov.br,"HTTPSConnectionPool(host='fazenda.gov.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face6251040>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='fazenda.gov.br', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7facdc103f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
fbr.gov.pk,"HTTPSConnectionPool(host='fbr.gov.pk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='fbr.gov.pk', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7face4f27fa0>, 'Connection to fbr.gov.pk timed out. (connect timeout=2)'))"
fda.gov,"HTTPSConnectionPool(host='fda.gov', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face07bf940>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='fda.gov', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face440ee20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
fdl.com.bd,"HTTPSConnectionPool(host='fdl.com.bd', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7facdc00fe20>, 'Connection to fdl.com.bd timed out. (connect timeout=2)'))","HTTPConnectionPool(host='fdl.com.bd', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7face6948490>, 'Connection to fdl.com.bd timed out. (connect timeout=2)'))"
felix.si,"HTTPSConnectionPool(host='felix.si', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='felix.si', port=80): Read timed out. (read timeout=2)"
femcafe.hu,"HTTPSConnectionPool(host='www.femcafe.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.femcafe.hu', port=443): Read timed out. (read timeout=2)"
femina.co.id,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
fero-term.hr,"HTTPSConnectionPool(host='fero-term.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.fero-term.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ffos.hr,"HTTPSConnectionPool(host='ffos.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7facdad92c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ffos.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7facde46a0d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
fhuce.edu.uy,"HTTPSConnectionPool(host='fhuce.edu.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face35d57c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='fhuce.edu.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face569c160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ficohsa.com,"HTTPSConnectionPool(host='ficohsa.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7face643b5b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ficohsa.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7face444d6a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
fiercebook.com,"HTTPSConnectionPool(host='fiercebook.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='fiercebook.com', port=443): Read timed out. (read timeout=2)"
fiksuruoka.fi,"HTTPSConnectionPool(host='fiksuruoka.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='fiksuruoka.fi', port=443): Read timed out. (read timeout=2)"
film-tv.at,"HTTPSConnectionPool(host='www.film-tv.at', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.film-tv.at', port=443): Read timed out. (read timeout=2)"
filmmodu.org,"HTTPSConnectionPool(host='filmmodu.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'filmmodu.org' doesn't match 'www.filmmodu.org'"")))","HTTPSConnectionPool(host='filmmodu.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'filmmodu.org' doesn't match 'www.filmmodu.org'"")))"
filmovamista.cz,"HTTPSConnectionPool(host='filmovamista.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='filmovamista.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
fimas.cl,"HTTPSConnectionPool(host='fimas.cl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='fimas.cl', port=443): Read timed out. (read timeout=2)"
finances.gov.ma,"HTTPSConnectionPool(host='finances.gov.ma', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7face696c4c0>, 'Connection to finances.gov.ma timed out. (connect timeout=2)'))","HTTPConnectionPool(host='finances.gov.ma', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7face5fa62e0>, 'Connection to finances.gov.ma timed out. (connect timeout=2)'))"
financialmirror.com,"HTTPSConnectionPool(host='www.financialmirror.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.financialmirror.com', port=443): Read timed out. (read timeout=2)"
